test/tests/api/v2_0/nodes_tests.py

```diff
@@ -283,6 +283,7 @@ def test_node_workflows_get(self):
                 resps.append(self.__get_data())
         for resp in resps:
             assert_not_equal(0, len(resp), message='No Workflows found for Node')
+        Api().nodes_get_workflow_by_id('fooey')
```

Comment:
@lacarb : Since no assertion after the API calling, so my understanding is that this line can be directly removed, not need to recover it.